packages/editor/src/components/entities-saved-states/entity-record-item.js

```diff
@@ -35,15 +36,31 @@ export default function EntityRecordItem( { record, checked, onChange } ) {
 	);
 
 	return (
-		<PanelRow>
-			<CheckboxControl
-				__nextHasNoMarginBottom
-				label={
-					decodeEntities( entityRecordTitle ) || __( 'Untitled' )
-				}
-				checked={ checked }
-				onChange={ onChange }
-			/>
-		</PanelRow>
+		<>
+			<PanelRow>
+				<CheckboxControl
+					__nextHasNoMarginBottom
+					label={
+						decodeEntities( entityRecordTitle ) || __( 'Untitled' )
+					}
+					checked={ checked }
+					onChange={ onChange }
+				/>
+			</PanelRow>
+			{ record.hasMetaChanges && (
+				<PanelRow>
+					<Flex className="entities-saved-states__block-bindings">
+						<Icon
+							className="entities-saved-states__connections-icon"
+							icon={ connection }
+							size={ 24 }
+						/>
+						<span className="entities-saved-states__bindings-text">
+							{ __( 'Block Bindings.' ) }
+						</span>
+					</Flex>
+				</PanelRow>
```

Comment:
I don't know if the message we show should be "Block Bindings". To me, it seems related to just Post Meta, not Block Bindings. If I am not mistaken, any change made to custom fields (with or without block bindings) will trigger this message, right?

Reply:
I had the same observation. I don't think we track, nor should we even try to do so, whether an edit was triggered by the established Block Binding. It's more about bringing more detailed information about what has been edited, similar to when folks edit the site settings: 

<img width="277" alt="Screenshot 2024-05-27 at 10 47 14" src="https://github.com/WordPress/gutenberg/assets/699132/a5bba9e2-7f3c-4006-8129-c5a7f143f3ae">

Reply:
So should we rename the message to "Post Meta" and remove the icon? cc @jasmussen

Reply:
Maybe, we should flex here to meet the beta deadline to be sure. I do like the icon, and I do think it's important to have some indentation, _or_ space below each post+post-meta chunk, so it's visibly clear that "post meta" is not some single unit you can save globally, it's attached and related to each post.

Reply:
The tricky thing with 'Post meta' is that the scope is wider than block bindings. 

For instance why would a 'Post meta' item be surfaced in the save panel when a block binding is changed, but not when something like the slug, template, or discussion settings are changed?